src/panels/lovelace/header-footer/hui-graph-header-footer.ts

```diff
@@ -112,8 +112,9 @@ export class HuiGraphHeaderFooter extends LitElement
   static get styles(): CSSResult {
     return css`
       .spinner {
-        height: 58px;
-        text-align: center;
+        display: flex;
+        height: 98px;
```

Comment:
The height is not fixed, it is 1/5 of the width.

Reply:
You can work with `margin-bottom: 20%`

Reply:
Ah OK. I'll fix now